jupyterlab/labapp.py

```diff
@@ -536,8 +537,32 @@ class LabApp(NBClassicConfigShimMixin, LabServerApp):
         help="Whether to expose the global app instance to browser via window.jupyterlab",
     )
 
-    collaborative = Bool(
-        False, config=True, help="Whether to enable collaborative mode (experimental)."
+    collaborative = Bool(False, config=True, help="Whether to enable collaborative mode.")
+
+    collaborative_file_poll_interval = Int(
+        1,
+        config=True,
+        help="""The period in seconds to check for file changes in the back-end (relevant only
```

Comment:
I am not quite sure what this means. Does this check to see if the file has changed on disk?

Reply:
Yes, if a file is changed "in the backend" (the result of a `git checkout branch`, for instance), this is seen as a Y update.

Reply:
I would phrase this as "changes on disk" rather than "backend"